.gitignore

```diff
@@ -1,6 +1,8 @@
 node_modules/
 /test/awesome-json-results/
 npm-debug.log
+yarn.lock
+shrinkwrap.yaml
```

Comment:
Why would we add `yarn.lock` if it's removed?  And what's `shrinkwrap.yaml`?

Reply:
shrinkwrap.yaml is what happens if you `pnpm i`.
It's removed because it is currently already in the repo, and it's added to `.gitignore` because contributors using `yarn` or `pnpm i` will create these and may easily commit them again.
(which they would be absolutely right to do if the repository's source of truth was `pnpm`'s `shrinkwrap.yaml` or `yarn`'s `yarn.lock` rather as the case seems to be `npm`'s `package-lock.json` )

Reply:
I see.  Thank you.